### ISSUE TITLE
MacOS version does not understand home folder as ~

### ISSUE THREAD

**Issue by nhanledev** [kingToolbox/WindTerm#872]
I use Windows, Linux and MacOS version of Windterm 2.4.1 using the same `.wind` folder.

On windows and Linux, Winterm can understand that `~` in `~/.ssh/mykey` is the home folder, but on MacOS, it does not recognize the `~` path.

Example is shown in the screenshot below. I think there is something wrong with parsing the home path on Mac.

<img width="549" alt="Screen Shot 2022-07-09 at 22 43 00" src="https://user-images.githubusercontent.com/28459250/178112833-85636b50-85f3-4a5c-a8b5-920287cbd9af.png">

**Comment by nhanledev** [comment]
The bug is still occurs on MacOS on version 2.5.0.
For more details:
- My ssh private key is at `Users/nhanle/.ssh/id_ed25519_nopass` on MacOS and `C:\Users\nhanle\.ssh\id_ed25519_nopass`
- I share the same `.wind` folder for both Mac and Windows
- I use the path `~/.ssh/id_ed25519_nopass` for my key
- On windows, I can establish connection to the destination server, on Mac I can not.
- MacOS only  recognize the file using `Users/nhanle/.ssh/id_ed25519_nopass`, not `~/.ssh/id_ed25519_nopass`

**Comment by xieccc** [comment]
Yes, it still exists.

**Comment by PMExtra** [comment]
Same problem here.
Did you find any solution?

**Comment by nhanledev** [comment]
No, looks like the author is not having the patch for this issue yet.

**Comment by PMExtra** [comment]
@kingToolbox Watch this issue again, please
